### PR TITLE
Create artifacts to given directory

### PR DESCRIPTION
**Pull request by chxchx**
Add a new flag to `release/create_release_archives.sh` so tar files are made in the given directory, which makes release automation easier as per https://github.com/istio/test-infra/pull/393

**Release note**:

```release-note
None
```

### CODE REVIEW

**Comment by chxchx** [comment]
/assign @rshriram
/assign @sebastienvas 
PTAL

File: release/create_release_archives.sh [hunk 1]
```diff
@@ -29,9 +29,16 @@ ISTIO_VERSION="$(cat ${ROOT}/istio.RELEASE)"
 [[ -z "${ISTIO_VERSION}" ]] && error_exit 'ISTIO_VERSION is not set'
 [[ -z "${ISTIOCTL_URL}" ]] && error_exit 'ISTIOCTL_URL is not set'
 
-TMP_DIR="$(mktemp -d /tmp/istio.version.XXXX)"
-COMMON_FILES_DIR="${TMP_DIR}/istio/istio-${ISTIO_VERSION}"
-ARCHIVES_DIR="${TMP_DIR}/archives"
+BASE_DIR="$(mktemp -d /tmp/istio.version.XXXX)"
```
Comment (sebastienvas) [inline release/create_release_archives.sh]: Here you are creating a temp dir. You should created it only if value is not net.

File: release/create_release_archives.sh [hunk 2]
```diff
@@ -29,9 +29,16 @@ ISTIO_VERSION="$(cat ${ROOT}/istio.RELEASE)"
...
+
+while getopts :d: arg; do
+  case ${arg} in
+    d) BASE_DIR="${OPTARG}";;
+  esac
+done
+
```
Comment (sebastienvas) [inline release/create_release_archives.sh]: Here you would add
[[ -z ${BASE_DIR} ]] && BASE_DIR="$(mktemp -d /tmp/istio.version.XXXX)"

File: release/create_release_archives.sh (hunk 1 above)
Comment (chxchx) [inline release/create_release_archives.sh]: Indeed. I overlooked the fact that a directory is actually created besides returning a path. Good catch.

File: release/create_release_archives.sh (hunk 2 above)
Comment (chxchx) [inline release/create_release_archives.sh]: Done

**Comment by chxchx** [comment]
PTAL

**Comment by chxchx** [comment]
/assign @rshriram
PTAL

**Pull request merged**